handler/transport.go

```diff
@@ -36,25 +38,36 @@ type transport struct {
 	cfg         config.Proxy
 	noAuthPaths []*policy.Assertion
 	// List to check for deprecated cipher suites
-	insecureCipherSuites []*tls.CipherSuite
+	insecureCipherSuites   []*tls.CipherSuite
+	latencyInstrumentation prometheus.Histogram
```

Comment:
remove prometheus dependency
```suggestion
	metrics Metrics
```

Reply:
@WindzCUHK fixedin https://github.com/AthenZ/authorization-proxy/pull/30/commits/1a0bc8cef3624f605e59ddad85deec897964644b